src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -599,66 +594,49 @@ public ResponseEntity<ExamExerciseStartPreparationStatus> getExerciseStartStatus
      * @param studentExam the student exam to be prepared
      */
     private void prepareStudentExamForConduction(HttpServletRequest request, User currentUser, StudentExam studentExam) {
-        examService.loadQuizExercisesForStudentExam(studentExam);
 
-        // 2nd: mark the student exam as started
-        studentExam.setStarted(true);
-        if (studentExam.getStartedDate() == null) {
-            studentExam.setStartedDate(ZonedDateTime.now());
+        // In case the studentExam is not yet started, a new participation wit a specific initialization date should be created - isStarted uses Boolean
```

Comment:
```suggestion
        // In case the studentExam is not yet started, a new participation with a specific initialization date should be created - isStarted uses Boolean
```

Reply:
thx, I'll fix it